README.md

```diff
@@ -161,7 +161,9 @@ There currently isn't an NI organization wide requirement to enforce accessibili
 A project should strive to adopt this configuration as fully as possible, but there are valid reasons to disable a rule across a codebase or specific directory:
 1. As a temporary measure to stage adoption of the tooling.
 2. As a permanent measure if the rule is incompatible with a project's configuration. The rule configuration files in this package (`index.js`, `typescript.js`, etc) contain comments on each rule if it might commonly be disabled. Some examples include:
-   - consider disabling `@typescript-eslint/no-floating-promises` in test code for Angular projects because it is incompatible with the `jasminewd2` library
+   - consider disabling the following rules for test code that uses `jasminewd2` because of an incompatibility and method spies
+     - `@typescript-eslint/no-floating-promises`
+     - `@typescript-eslint/unbound-method`
```

Comment:
Could you provide an example that shows why unbound-method doesn't work with jasminewd2? (Doesn't have to be in the README for now, just as a reply or in the PR description).

Reply:
This can be found in the examples provided with the [documentation on spies](https://jasmine.github.io/2.0/introduction#section-Spies).
```typescript
expect(foo.setBar).toHaveBeenCalled();
```